qa/automation/ScoringPortal.Tests.Scenarios/Sign in with Google account.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 1. Sign in with Google Account
+
+   Background:
+      Given the user has a Google account
+
+   Scenario: 1.1. The application link can be opened
+      Given user has a link for the application
+      When user clicks on the link
```

Comment:
I think in background the info about installed vpn might be added. Because If I try to execute this on my computer without vpn, it will not work.